pkg/kubernetes/api/sloth/v1/types.go

```diff
@@ -68,6 +68,9 @@ type SLO struct {
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
 
+	// Info labels added to the `sloth_slo_info` metric
+	InfoLabels map[string]string `json:"info_labels,omitempty"`
```

Comment:
IIRC, Kubernetes makes you do camelCase'ing. See below like the `errorRatioQuery` field

Reply:
Confirmed: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions

> Go field names must be PascalCase. JSON field names must be camelCase. Other than capitalization of the initial letter, the two should almost always match. No underscores or dashes in either.

Reply:
fixed in [87bb5a2](https://github.com/linode-obs/sloth/pull/4/commits/87bb5a228e0dfa6b7a8488abdd2014b6c3149ef5), thanks